docs/themes/porter/layouts/index.html

```diff
@@ -51,41 +51,30 @@ <h1>
+                  to deploy it: command-line tools, configuration files, secrets, and bash scripts to glue it all together. 
+                  Package that into a versioned bundle distributed over standard Docker registries or plain tgz files.</p>
+                <p>Now anyone can install your application without deep knowledge of your deployment process,
+                   or following a step-by-step deployment doc, regardless of the tech stack.
+                </p>
 
-                <p>When we deploy to the cloud, most of us aren’t dealing with just a single cloud provider or toolchain.
-                  The simplest of applications today need a load balancer, SSL certificate, persistent file storage, DNS, and
-                  somewhere in there is your application. One app is installed with Helm, another with the cloud provider’s cli
-                  and it is all glued together with magic bash scripts.
-
-                <p>That is a lot to figure out! 😅</p>
-
-                <p>Porter is a cloud installer based on the Cloud Native Application Bundle (<a href="https://cnab.io">CNAB</a>)
-                  spec that helps you manage everything together in a single bundle, focusing on what you know best: your application.</p>
-              </div>
-
-              <div class="hide-for-small-down medium-6 columns">
-                <img src="{{ .Site.BaseURL }}images/porter-mixin-cloud.png" alt="Porter Mixin Cloud" class="helm-overview-img">
-              </div>
-            </div>
-            <div class="row">
-              <div class="small=1 columns">
-                  <h2>When would I use a bundle?</h2>
-
-                  <ul>
-                    <li>Install the tools to manage your application: helm, aws/azure/gcloud, terraform</li>
-                    <li>Deploy an application along with its infrastructure: cloud storage, dns entry, load balancer, ssl cert</li>
-                    <li>Get software and its dependencies into airgapped networks</li>
-                    <li>Manage disparate operational tech, such as Helm, Chef, or Terraform, across teams and departments</li>
-                  </ul>
+                <h2>Why Porter?</h2>
+                <ul>
+                  <li>Single command to find and deploy any application, regardless of the tech stack. No knowledge of the deployment process necessary.</li>
+                  <li>Secure handling of secrets and credentials, integrated with industry standard secret stores.</li>
+                  <li>Share the operational load of managing your team's deployments   .</li>
+                  <li>Works with and improves existing tools, such as bash, cloud CLIs, terraform and helm. You don't rewrite your existing deployments to start using Porter.</li>
+                  <li>Reliable deployments because the client environment, for example the version of version of terraform or script parameters provided, is consistent and enforced.</li>
```

Comment:
```suggestion
                  <li>Reliable deployments because the client environment, for example the version of terraform or script parameters provided, is consistent and enforced.</li>
```